<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackendConfiguration</class>
 <widget class="QDialog" name="BackendConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translation Services</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="CListWidget" name="listWidget_backends">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new_backend">
       <property name="text">
        <string>Add New API</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_icon">
         <property name="text">
          <string notr="true">&lt;icon&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>12</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_backend_name">
         <property name="text">
          <string notr="true">&lt;backend name&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cost_icon">
         <property name="text">
          <string>&lt;cost icon&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_reliability_icon">
         <property name="text">
          <string notr="true">&lt;reliability icon&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_description">
       <property name="text">
        <string notr="true">*description*</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="BackendSettings" name="backend_settings">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>406</width>
          <height>362</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BackendSettings</class>
   <extends>QWidget</extends>
   <header>deepqt.backend_settings</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CListWidget</class>
   <extends>QListWidget</extends>
   <header>deepqt.CustomQ.CListWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BackendConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BackendConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
